data/mods/DinoMod/monsters/dinosaur.json

```diff
@@ -131,24 +131,25 @@
     "weight": "1000 kg",
     "bodytype": "bear",
     "material": [ "flesh" ],
-    "aggression": 100,
-    "morale": 90,
-    "speed": 100,
+    "//": "Altering Spinosaurus to reflect modern fossils and research on speed, bite force, teeth shape, tail shape and size as well changing its aggression and morale to reflect our understanding of eating and hunting behaviours.",
```

Comment:
You probably want to add those fossils/research to the PR description.

Reply:
done